### PR TITLE
Use originalLogin if username empty

### PR DESCRIPTION
**Pull request by mrajashree**
https://github.com/rancher/rancher/issues/10189

### CODE REVIEW

**Comment by prachidamle** [comment]
If token cookie is missing in the API request, what does Cattle GET /v2-beta/token return?
If token cookie is missing, can we avoid the extra request to cattle?

**Comment by mrajashree** [comment]
@prachidamle changed the code to avoid GET request to v2-beta/token if no token is found in cookies

File: server/auth_server.go [hunk 1]
```diff
@@ -1089,36 +1090,38 @@ func TestLogin(testAuthConfig model.TestAuthConfig, accessToken string, token st
...
-	req, err := http.NewRequest("GET", getURL, nil)
-	if err != nil {
-		return 0, fmt.Errorf("Error %v in getting token", err)
-	}
-	req.Header.Set("Cookie", "token="+token)
+		req, err := http.NewRequest("GET", getURL, nil)
+		if err != nil {
+			return 0, fmt.Errorf("Error %v in getting token", err)
+		}
+		req.Header.Set("Cookie", "token="+token)
```
Comment (prachidamle) [inline server/auth_server.go:1104]: Are there tabs here, can we check the formatting?

**Comment by prachidamle** [comment]
LGTM

File: server/auth_server.go (hunk 1 above)
Comment (prachidamle) [reply server/auth_server.go:1104]: As long as go fmt is happy with above, we should be good.

**Pull request merged**